chainerrl/policies/gaussian_policy.py

```diff
@@ -145,13 +145,20 @@ class FCGaussianPolicyWithStateIndependentCovariance(
             'spherical' or 'diagonal'.
         nonlinearity (callable): Nonlinearity placed between layers.
         mean_wscale (float): Scale of weight initialization of the mean layer.
+        var_func (callable): Callable that computes the variance from the var
+            parameter. It should always return positive values.
+        var_param_init (float): Initial value the var parameter.
     """
 
     def __init__(self, n_input_channels, action_size,
                  n_hidden_layers=0, n_hidden_channels=None,
                  min_action=None, max_action=None, bound_mean=False,
                  var_type='spherical',
-                 nonlinearity=F.relu, mean_wscale=1):
+                 nonlinearity=F.relu,
+                 mean_wscale=1,
+                 var_func=F.softplus,
+                 var_param_init=0,
```

Comment:
Do you think the name `var_param_init` suggests the correspondence to the names `var_wscale` and `var_bias` of `FCGaussianPolicy`?

Reply:
Maybe `var` is more consistent with `var_bias`, while less informative. Do you think `var` is better?

Reply:
Actually, this param does not represent variance, but it represents values that are converted to variance via `var_func`, so I added `_param`. I admit it is still confusing, but I didn't come up with a better name. Any suggestion?